ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/port-channel-interfaces.cfg

```diff
@@ -53,6 +53,8 @@ interface Port-Channel10
    evpn ethernet-segment
       identifier 0000:0000:0404:0404:0303
       route-target import 04:04:03:03:02:02
+   spanning-tree bpduguard disable
+   spanning-tree bpdufilter disable
```

Comment:
Also generated different in AVD than EOS:
```
hss044(config)#sh run int po10
interface Port-Channel10
   description SRV01_bond0
   switchport trunk allowed vlan 2-3000
   switchport mode trunk
   !
   evpn ethernet-segment
      identifier 0000:0000:0404:0404:0303
      route-target import 04:04:03:03:02:02
   shape rate 50 percent
   spanning-tree bpduguard disable
   spanning-tree bpdufilter disable
```

Reply:
I will move shape.rate above bpduguard.